redis-snapshot-replicator/README.md

```diff
@@ -0,0 +1,27 @@
+TODO manual
+
+Elasticache needs to have account permissions to the S3 bucket, without this explicit access, it is unable to store backups in S3. In this link, there is more info : https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/backups-exporting.html#backups-exporting-grant-access
+
+These are the steps you have to take
```

Comment:
Can you add a bit of structure to this readme? Like a numbered list for example. Otherwise it's very confusing